packages/ui/src/views/execution/pipeline-status.tsx

```diff
@@ -1,67 +1,23 @@
-import { FC, useEffect, useRef, useState } from 'react'
-
 import { ExecutionState } from '@views/repo/pull-request'
 
 import { ExecutionStatus } from './execution-status'
 
-interface PipelineStatusProps {
+const PipelineStatus = ({
```

Comment:
looks like the interface got removed, and we have inline types now.. 
not a blocker, but let's come back to it later